test.js

```diff
@@ -302,4 +464,4 @@ if ( ! process.browser) {
       t.end();
     });
   });
-}
+}
```

Comment:
Thanks for the good test additions.

Reply:
I mean I doubled the run time of the test, but tests need to be done, right?